packages/desktop/package.json

```diff
@@ -124,6 +124,7 @@
     "@redux-saga/types": "^1.1.0",
     "@reduxjs/toolkit": "^1.6.2",
     "@sentry/electron": "^2.5.4",
+    "@sentry/browser": "^6.19.7",
```

Comment:
I assume that you didn't manage to update package-lock

Reply:
you're right